index.md

```diff
@@ -1,11 +1,58 @@
 # TMA Performance Settings vs Capacitor Bank
 
 ```{abstract}
-We want to compare the maximum velocity, acceleration and jerk on every slew with the minimum value of the capacitor banks on the TMA.
-
-Phase, the contractor responsible for the TMA Power Supply and the Capacitor Banks, suggests that we keep these voltage drops above 575 V.
+This tech note presents an analysis of the the maximum velocity, acceleration and jerk on every slew with the minimum value of the capacitor banks on the TMA. This is in support of [SITCOM-1146](https://jira.lsstcorp.org/browse/SITCOM-1146).
 ```
 
-## Add content here
+## Introduction
+The contractor responsible for the TMA Power supply and capacitor banks, **_Phase_**, has indicated that it is best to keep the voltage drops during a slew to above 575 V. 
+We looked at the data from several nights of testing in Dec. 2023 to evalute the relationship between the voltage drop during a slew and the performance of the TMA during that slew. 
+The data analysed was taken as part of [BLOCK-160](https://jira.lsstcorp.org/browse/BLOCK-160) and [BLOCK-164](https://jira.lsstcorp.org/browse/BLOCK-164), covering tests performed December 11 - 15, 2023.
+
+In this analysis, we identified each slew during these nights of observing using `TMAEventMaker()`.
+Then we queried the EFD for data taken during those slews from the following topics
+* `lsst.sal.MTMount.mainPowerSupply`
+* `lsst.sal.MTMount.elevation`
+* `lsst.sal.MTMount.azimuth` 
+
+Using the method defined in [SITCOMTN-067](https://sitcomtn-067.lsst.io), we calculated the acceleration and jerk from the measured velocity and then found the maximum value during a slew. 
+
+This analysis can be found at https://github.com/lsst-sitcom/notebooks_vandv/tree/develop/notebooks/tel_and_site/subsys_req_ver/tma/SITCOMTN-110_TMA_Perf_vs_Capacitor_Banks.ipynb.
+
+
+## Results
+### Overall
+Here are the histograms for the data from all slews in BLOCK-160/164. There were ~1500 slews during those days of observation.
+
+![](images/min_power_all_slews.png)
```

Comment:
Can you explain a bit more what the "Minimum TMA Power per Slew" means? Is this the drop in the power supply?

Reply:
Added this line to introduction: "In this analysis, we compare the maximum values of the performance of the TMA to the minimum value recorded during the slew of the `powerSupplyVoltage` for the `mainPowerSupply`."